apps/builddao/widget/components/modals/propose/RemoveMember.jsx

```diff
@@ -167,6 +176,38 @@ const TextareaWrapper = styled.div`
   }
 `;
 
+useEffect(() => {
+  let timeoutId;
+
+  if (copied) {
+    sdk.createRemoveMemberProposal({
+      description: text,
+      memberId: accountId,
+      roleId: role,
+      gas: 180000000000000,
+      deposit: 200000000000000,
+      additionalCalls: notificationsData,
+    });
+
+    timeoutId = setTimeout(() => {
+      setCopied(false);
+    }, 2000);
+  }
+
+  return () => clearTimeout(timeoutId);
+}, [copied]);
+
+const handleCopy = () => {
+  clipboard
```

Comment:
ahh so copy doesn't work from info widget?

Reply:
no actually I got to know that we cant use state and then import it through VM

Reply:
why don't we have this inside the InfoPopup component?
because we need it for all modals right?